Matrix/src/Matrix.cpp

```diff
@@ -105,6 +105,23 @@ Matrix &Matrix::operator+(Matrix &mat_1)
 	return mat_1;
 }
 
+Matrix &Matrix::operator+=(const Matrix& RHS)
+{
```

Comment:
What happens if matrices have different sizes? Lets say:

    Matrix A(3,3);
    Matrix B(4,4);
    A+=B;

Would this work ?